src/lando/main/scm/helpers.py

```diff
@@ -129,6 +129,13 @@ def get_timestamp_from_hg_date_header(date_header: str) -> str:
 class PatchHelper(ABC):
     """Base class for parsing patches/exports."""
 
+    # A handy regex to match a From header containing user details
+    USERNAME_RE = re.compile(r"^From:\s*(?P<user>(?P<name>.*[^\s])?\s*<(?P<email>.*)>)")
```

Comment:
I'm noticing a lot of regex strings in recent PRs. I wonder if we should have them all in one module and import them as needed.

Reply:
This one's gone now, replaced by the `PullRequest.author` logic.